config/locales/user_preferences/en.yml

```
@@ -0,0 +1,3 @@
+en:
+  user_preferences:
+    auto_update_exercises: "Automatically update my solutions to the latest versions of exercises if they pass the latest tests"
```

I'm not entirely sure, but should this be: 

```suggestion
    auto_update_exercises: "Automatically update my solutions to the latest version of exercises if they pass the latest tests"
```

or maybe even:

```suggestion
    auto_update_exercises: "Automatically update my solutions to the latest version of exercises if they pass the latest version's tests"
```